src/tests/ftest/launch.py

```diff
@@ -1724,6 +1722,8 @@ def install_debuginfos():
         on this node also.
 
     """
+    from distro_utils import detect     # pylint: disable=import-outside-toplevel
```

Comment:
It's also possible to import like this:
```
try:
    from distro_utils import detect
except:
    detect = None
```

Reply:
Yes, thanks. This is really temporary until CORCI-1100 is fixed.  We could do this and add better handling of this situation or when `detect()` returns `UNKNOWN_DISTRO_NAME`.  I plan to address issues like this as part of DAOS-9828.